diff --git a/node_modules/@achingbrain/ssdp/dist/src/default-ssdp-options.js b/node_modules/@achingbrain/ssdp/dist/src/default-ssdp-options.js
index 41ac65e..8c63670 100644
--- a/node_modules/@achingbrain/ssdp/dist/src/default-ssdp-options.js
+++ b/node_modules/@achingbrain/ssdp/dist/src/default-ssdp-options.js
@@ -1,9 +1,13 @@
 import { webcrypto as crypto } from 'crypto'; // remove when having crypto global
-import { createRequire } from 'module';
+// import { createRequire } from 'module';
 import mergeOptions from 'merge-options';
 import { defaultSocketOptions } from './default-socket-options.js';
-const req = createRequire(import.meta.url);
-const { name, version } = req('../../package.json');
+// const req = createRequire(import.meta.url);
+// const { name, version } = req('../../package.json');
+const {name, version} = {
+    name: "@achingbrain/ssdp",
+    version: "4.0.1"
+};
 const DEFAULT_SSDP_SIGNATURE = `node.js/${process.version.substring(1)} UPnP/1.1 ${name}/${version}`;
 export function defaultSsdpOptions(options) {
     return mergeOptions(options ?? {}, {
