diff --git a/node_modules/@holochain/client/lib/api/zome-call-signing.js b/node_modules/@holochain/client/lib/api/zome-call-signing.js
index aaddf61..fa20172 100644
--- a/node_modules/@holochain/client/lib/api/zome-call-signing.js
+++ b/node_modules/@holochain/client/lib/api/zome-call-signing.js
@@ -1,5 +1,7 @@
 import nacl from "tweetnacl";
 import { encodeHashToBase64 } from "../utils/base64.js";
+import crypto from 'node:crypto';
+
 const signingCredentials = new Map();
 /**
  * Get credentials for signing zome calls.
@@ -54,7 +56,6 @@ export const randomByteArray = async (length) => {
         return window.crypto.getRandomValues(new Uint8Array(length));
     }
     else {
-        const crypto = await import("crypto");
         return new Uint8Array(crypto.randomBytes(length));
     }
 };
