diff --git a/lib/source/node.js b/lib/source/node.js
index 391749f91d15ec733f3db8f15205474001bbc5b9..302627d6846537f7853f22f5bfe232b507bec2fd 100644
--- a/lib/source/node.js
+++ b/lib/source/node.js
@@ -1,4 +1,5 @@
 "use strict";
+import nodeCrypto from "node:crypto";
 // Copyright (C) 2016 Dmitry Chestnykh
 // MIT License. See LICENSE file for details.
 Object.defineProperty(exports, "__esModule", { value: true });
@@ -9,7 +10,6 @@ class NodeRandomSource {
         this.isAvailable = false;
         this.isInstantiated = false;
         if (typeof require !== "undefined") {
-            const nodeCrypto = require("crypto");
             if (nodeCrypto && nodeCrypto.randomBytes) {
                 this._crypto = nodeCrypto;
                 this.isAvailable = true;
