diff --git a/node_modules/@transmute/did-key.js/dist/index.js b/node_modules/@transmute/did-key.js/dist/index.js
index a21226a..8a3a30e 100644
--- a/node_modules/@transmute/did-key.js/dist/index.js
+++ b/node_modules/@transmute/did-key.js/dist/index.js
@@ -1,7 +1,7 @@
 
 'use strict'
 
-if (process.env.NODE_ENV === 'production') {
+if (true) {
   module.exports = require('./did-key.js.cjs.production.min.js')
 } else {
   module.exports = require('./did-key.js.cjs.development.js')
