diff --git a/node_modules/@transmute/web-crypto-key-pair/dist/index.js b/node_modules/@transmute/web-crypto-key-pair/dist/index.js
index 99c9cf9..81db4a1 100644
--- a/node_modules/@transmute/web-crypto-key-pair/dist/index.js
+++ b/node_modules/@transmute/web-crypto-key-pair/dist/index.js
@@ -1,7 +1,7 @@
 
 'use strict'
 
-if (process.env.NODE_ENV === 'production') {
+if (true) {
   module.exports = require('./web-crypto-key-pair.cjs.production.min.js')
 } else {
   module.exports = require('./web-crypto-key-pair.cjs.development.js')
