diff --git a/node_modules/hypercore/index.js b/node_modules/hypercore/index.js
index 7ad78f6..7280801 100644
--- a/node_modules/hypercore/index.js
+++ b/node_modules/hypercore/index.js
@@ -1,4 +1,4 @@
-const { EventEmitter } = require('events')
+import { EventEmitter } from 'node:events'
 const RAF = require('random-access-file')
 const isOptions = require('is-options')
 const hypercoreCrypto = require('hypercore-crypto')
diff --git a/node_modules/hypercore/lib/batch.js b/node_modules/hypercore/lib/batch.js
index 37bcafe..5ebc716 100644
--- a/node_modules/hypercore/lib/batch.js
+++ b/node_modules/hypercore/lib/batch.js
@@ -1,5 +1,5 @@
 const { BLOCK_NOT_AVAILABLE, SESSION_CLOSED } = require('hypercore-errors')
-const EventEmitter = require('events')
+import { EventEmitter } from 'node:events'
 const c = require('compact-encoding')
 const b4a = require('b4a')
 
