diff --git a/node_modules/inherits/inherits.js b/node_modules/inherits/inherits.js
index f71f2d9..c973372 100644
--- a/node_modules/inherits/inherits.js
+++ b/node_modules/inherits/inherits.js
@@ -1,5 +1,5 @@
 try {
-  var util = require('util');
+  var util = require('node:util');
   /* istanbul ignore next */
   if (typeof util.inherits !== 'function') throw '';
   module.exports = util.inherits;
