diff --git a/node-gyp-build.js b/node-gyp-build.js
index 61b398efcbf3d504a8a2f8ecfdf1c21481cc3243..602cce41bb105890126bb3384bc64a4fce3805b6 100644
--- a/node-gyp-build.js
+++ b/node-gyp-build.js
@@ -1,6 +1,6 @@
-var fs = require('fs')
-var path = require('path')
-var os = require('os')
+import fs from 'node:fs'
+import path from 'node:path'
+import os from 'node:os'
 
 // Workaround to fix webpack's build warnings: 'the request of a dependency is an expression'
 var runtimeRequire = typeof __webpack_require__ === 'function' ? __non_webpack_require__ : require // eslint-disable-line
