diff --git a/node_modules/safe-buffer/index.js b/node_modules/safe-buffer/index.js
index f8d3ec9..56099bd 100644
--- a/node_modules/safe-buffer/index.js
+++ b/node_modules/safe-buffer/index.js
@@ -1,6 +1,6 @@
 /*! safe-buffer. MIT License. Feross Aboukhadijeh <https://feross.org/opensource> */
 /* eslint-disable node/no-deprecated-api */
-var buffer = require('buffer')
+import buffer from 'node:buffer';
 var Buffer = buffer.Buffer
 
 // alternative to using Object.keys for old browsers
