diff --git a/index.js b/index.js
index f8d3ec98852f449b44b7d89fc82bae737c69f3fc..56099bd9d40a6b0820901cb04731389008537d61 100644
--- a/index.js
+++ b/index.js
@@ -1,6 +1,6 @@
 /*! safe-buffer. MIT License. Feross Aboukhadijeh <https://feross.org/opensource> */
 /* eslint-disable node/no-deprecated-api */
-var buffer = require('buffer')
+import buffer from 'node:buffer';
 var Buffer = buffer.Buffer
 
 // alternative to using Object.keys for old browsers
