diff --git a/index.js b/index.js
index f8d3ec98852f449b44b7d89fc82bae737c69f3fc..57363e84b27073499c987e7a01f05c8a0034c4f2 100644
--- a/index.js
+++ b/index.js
@@ -1,6 +1,6 @@
 /*! safe-buffer. MIT License. Feross Aboukhadijeh <https://feross.org/opensource> */
 /* eslint-disable node/no-deprecated-api */
-var buffer = require('buffer')
+import buffer from 'node:buffer'
 var Buffer = buffer.Buffer
 
 // alternative to using Object.keys for old browsers
